<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>complement_recherche</class>
 <widget class="QWidget" name="complement_recherche">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Compléments d'informations:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Adresse complette de connection</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_adresse_complette"/>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_firefox">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Images/firefox.jpg</normaloff>Images/firefox.jpg</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_chrome">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Images/chrome.png</normaloff>Images/chrome.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_internet_explorer">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Images/internet_explorer.png</normaloff>Images/internet_explorer.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Autres options:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_ssh">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Images/ssh.ico</normaloff>Images/ssh.ico</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_telnet">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Images/telnet.jpeg</normaloff>Images/telnet.jpeg</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_putty">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Images/putty.png</normaloff>Images/putty.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_DM">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>DM</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Informations supplémentaires:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="complement"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
